### ISSUE TITLE
CurlWrapper not decoding compressed contents

### ISSUE THREAD

**Issue by tomascohen** [Aspen-Discovery/aspen-discovery#2135]
A recent change in Koha made compression of API requests actually work. After upgrading one of those Koha instances, we noticed API requests that get added the `Accept-Encoding: gzip, deflate` header stopped working.

After some digging and using the interactive debugger on my IDE, I understood compression hasn't really been working (i.e. Koha was returning raw text) and now that it works, the cURL wrapper is returning gzipped contents to the caller.

What we need is for the cURL wrapper to automatically decode the response contents.

**Comment by tomascohen** [comment]
I think we can close this now.

**Comment by mdnoble73** [comment]
Agreed that closing is best since GitHub issues do not seem to have any form of QA status or similar that we could use to indicate that it is done but not released yet.

**Comment by mdnoble73** [comment]
I don't think we need to change now since we're getting close to a decision on tool. Jira supports the workflows natively so if we go with that, anything we change here would likely not be needed.
